src/Akeneo/Pim/Enrichment/Component/Category/Connector/ArrayConverter/FlatToStandard/Category.php

```diff
@@ -17,12 +19,19 @@ class Category implements ArrayConverterInterface
     /** @var FieldsRequirementChecker */
     protected $fieldChecker;
 
+    /** @var LocaleRepositoryInterface */
+    protected $localeRepository;
+
     /**
      * @param FieldsRequirementChecker $fieldChecker
      */
-    public function __construct(FieldsRequirementChecker $fieldChecker)
-    {
+    public function __construct(
+        FieldsRequirementChecker $fieldChecker,
+        // TODO pull-up 6.0/master
```

Comment:
In case it's not you that made the pull up
```suggestion
        // TODO pull-up 6.0/master: Make this $variable not nullable
```